### PR TITLE
Add endpoint to change user permission

### PR DESCRIPTION
**Pull request by rtaieb**
An admin can now change a user role from "Travailleur mobile" to "Gestionnaire"

https://trello.com/c/KteDKFp7/483-etq-gestionnaire-je-peux-changer-les-droits-administrateurs-dun-salari%C3%A9-d%C3%A9j%C3%A0-rattach%C3%A9-%C3%A0-mon-entreprise

PR du front associée : 
https://github.com/MTES-MCT/mobilic/pull/124

### CODE REVIEW

File: app/controllers/employment.py [hunk 1]
```diff
@@ -476,3 +477,35 @@ def mutate(cls, _, info, employment_id):
...
+    class Arguments:
+        employment_id = graphene.Argument(
+            graphene.Int,
+            required=True,
+            description="Identifiant du rattachement pour lequel le rôle doit être changé",
+        )
+        has_admin_rights = graphene.Argument(
+            graphene.Boolean,
+            required=True,
+            description="Précise si le salarié rattaché est gestionnaire de l'entreprise, et s'il pourra donc avoir les droits de consultation et d'admnistration associés.",
```
Comment (tristan-gueguen) [inline app/controllers/employment.py:492]: administration *

File: app/controllers/employment.py [hunk 2]
```diff
@@ -476,3 +477,35 @@ def mutate(cls, _, info, employment_id):
...
+
+    @classmethod
+    @with_authorization_policy(
+        company_admin,
+        get_target_from_args=lambda *args, **kwargs: Employment.query.get(
+            kwargs["employment_id"]
+        ).company_id,
+        error_message="Actor is not authorized to change employee role",
+    )
+    def mutate(cls, _, info, employment_id, has_admin_rights):
```
Comment (tristan-gueguen) [inline app/controllers/employment.py:505]: Shouldn't we check if the user is an admin of the company of the employment ?

**Review by tristan-gueguen — COMMENTED** [review]
Maybe we could do a couple of unit tests ?
Check the error if we try to modify ourself, check we can't modify if we're not an admin of the relevant company

File: app/controllers/employment.py (hunk 2 above)
Comment (rtaieb) [reply app/controllers/employment.py:505]: This is already done with the _company_admin_ authorization policy.

**Review by tristan-gueguen — APPROVED** [review]

**Pull request merged**